packages/block-editor/src/store/selectors.js

```diff
@@ -560,18 +560,18 @@ export function getSelectedBlocksInitialCaretPosition( state ) {
 }
 
 /**
- * Returns the current multi-selection set of block client IDs, or an empty
- * array if there is no multi-selection.
+ * Returns the current selection set of block client IDs (multiselection or single selection).
  *
  * @param {Object} state Editor state.
  *
  * @return {Array} Multi-selected block client IDs.
  */
-export const getMultiSelectedBlockClientIds = createSelector(
+export const getSelectedBlockClientIds = createSelector(
 	( state ) => {
 		const { start, end } = state.blockSelection;
 		if ( start === end ) {
-			return [];
+			const selectedBlockClientId = getSelectedBlockClientId( state );
+			return selectedBlockClientId ? [ selectedBlockClientId ] : [];
```

Comment:
I guess we could safely use `EMPTY_ARRAY` as the fallback here? Same if we use my suggested variant above.